html/elements/input/password.json

```diff
@@ -0,0 +1,1070 @@
+                "edge_mobile": {
+                  "version_added": null
+                },
+                "firefox": {
+                  "version_added": "4"
+                },
+                "firefox_android": {
+                  "version_added": "4"
+                },
+                "ie": {
+                  "version_added": "10"
+                },
+                "opera": {
+                  "version_added": "11"
+                },
+                "opera_android": {
+                  "version_added": "12"
+                },
+                "safari": {
+                  "version_added": "5"
+                },
+                "safari_ios": {
+                  "version_added": "4"
+                },
+                "webview_android": {
+                  "version_added": null
+                }
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "readonly": {
+            "__compat": {
+              "description": "<code>readonly</code> attribute",
+              "support": {
+                "chrome": {
+                  "version_added": "1"
+                },
+                "chrome_android": {
+                  "version_added": null
+                },
+                "edge": {
+                  "version_added": null
+                },
+                "edge_mobile": {
+                  "version_added": null
+                },
+                "firefox": {
+                  "version_added": "1"
+                },
+                "firefox_android": {
+                  "version_added": true
+                },
+                "ie": {
+                  "version_added": "6",
+                  "notes": " Missing for <code>type='checkbox'</code> and <code>type='radio'</code>."
+                },
+                "opera": {
+                  "version_added": "10"
+                },
+                "opera_android": {
+                  "version_added": true
+                },
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": {
+                  "version_added": null
+                },
+                "webview_android": {
+                  "version_added": null
+                }
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "required": {
+            "__compat": {
+              "description": "<code>required</code> attribute",
+              "support": {
+                "chrome": {
+                  "version_added": "5",
+                  "notes": "Chrome 10 added support for <code><a href='https://developer.mozilla.org/docs/Web/HTML/Element/select'><select></a></code> element."
```

Comment:
`<select>` needs to be escaped here (i.e., `&lt;select&gt;`)

Reply:
I don't think this is relevant to password inputs, is it?